lib/rubocop/cop/style/inverse_methods.rb

```diff
@@ -166,6 +168,12 @@ def end_parentheses(node, method_call)
           method_call.source_range.end.join(node.source_range.end)
         end
 
+        def incompatible_method?(node)
+          return false if node.send_type?
+
+          INCOMPATIBLE_INVERSE_METHODS.include?(node.method_name)
+        end
```

Comment:
These are only incompatible with safe navigation, correct? I think it'd make sense to make the method name more descriptive.

Reply:
Finding a better name is difficult 😅  but `incompatible_context?` has been chosen.